extensions/wikia/CommunityPage/styles/components/_avatar.scss

```diff
@@ -16,6 +20,14 @@
 		background-position: center;
 		background-repeat: no-repeat;
 	}
+
+	&.more {
+		background: $avatar-number-background;
```

Comment:
Added separate PR as it cover bit wider changes https://github.com/Wikia/app/pull/10417